src/editor/action/action.cpp

```diff
@@ -260,11 +253,11 @@ editor_action* editor_action_starting_position::perform(map_context& mc) const
 		// action_chain.
 		editor_action_chain* undo_chain = new editor_action_chain();
```

Comment:
This line should use unique_ptr too, but both I (in #5260) and AI0867 missed it.  (I'm reviewing a diff between the two PRs.)